lib/binance/config.ex

```diff
@@ -0,0 +1,43 @@
+defmodule Binance.Config do
+  require Logger
+
+  @type t :: %Binance.Config{
+          api_key: String.t(),
+          api_secret: String.t()
+        }
+
+  @enforce_keys [:api_key, :api_secret]
+  defstruct [:api_key, :api_secret]
+
+  @doc """
+  Get default API configs
+
+  ## Examples
+      iex> Binance.Config.get()
+  """
+  def get(nil) do
+    %__MODULE__{
+      api_key: Application.get_env(:binance, :api_key),
+      api_secret: Application.get_env(:binance, :secret_key)
+    }
+  end
+
+  @doc """
+  Get dynamic API configs via ENVs
+
+  ## Examples
+      iex> Binance.Config.get(%{access_keys: ["B1_API_KEY", "B1_API_SECRET"]})
+  """
+  def get(%{
```

Comment:
Why does this have to be a map with a list? 

Can't we do something simpler like `get(api_key, api_secret)` or `get([api_key, api_secret])`?

Using a list with 2 fixed arguments also seem a bit weird, maybe a tuple would be better here? But I may be missing something.